addon/manifests/chart/templates/service.yaml

```diff
@@ -0,0 +1,26 @@
+---
+{{- if .Values.prometheus.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "controller.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: {{ template "controller.fullname" . }}
+    chart: {{ template "search.chart" . }}
+    release: {{ .Release.Name }}
+    search-monitor: {{ template "controller.fullname" . }}
+    heritage: {{ .Release.Service }}
+    addon.open-cluster-management.io/hosted-manifest-location: hosting
```

Comment:
I don't think this label is needed. I'm not sure what it does and we don't have it on the existing resources in this chart.